epoch/api/naming_system_api_usage.md

```diff
@@ -75,19 +104,25 @@ curl http://localhost:3013/v2/name\?name\=fooo.baar.aet
 {"name":"fooo.baar.aet","name_ttl":600000,"pointers":"{\"account_pubkey\":\"ak$3scLu3oJbhsdCJkDjfJ6BUPJ4M9ZZJe57CQ56deSbEXhaTSfG3Wf3i2GYZV6APX7RDDVk4Weewb7oLePte3H3QdBw4rMZw\"}"}
 ```
 
+### Spend to name
+
 Now you may use account and oracle pointers interchangeably with their addresses.
 In order to utilize account pubkey pointer, put a name instead of an account key, e.g. in spend transaction put it in `recipient_pubkey` field:
 ```
 curl -X POST -H "Content-Type: application/json" -d '{"recipient_pubkey":"fooo.baar.aet", "amount":2, "fee":1}' http://127.0.0.1:3113/v2/spend-tx
 {}
 ```
```

Comment:
I tested up to spend - with two nodes.
I posted the spend tx with API. Then I queried mempool with tx and I found not such tx there - probably it was already mined (I was using fast mining).
I enquired top for height - then I issues sequence of `curl 'http://127.0.0.1:3103/v2/block/txs/list/height?from=190&to=193&tx_encoding=json&tx_types=spend'` for finding the spend tx though I did not find that. This user experience hiccup is tracked as https://www.pivotaltracker.com/story/show/154871850